schema/cql/cql_test.go

```diff
@@ -71,7 +80,18 @@ create materialized view "i2" as
   where "int64type" is not null
   primary key (int64type, booltype ASC);`,
 		},
-		// TODO: Add more test cases
```

Comment:
We no longer need to add more test cases?

Reply:
kk. put it back